api/views/events.py

```diff
@@ -20,8 +20,10 @@ def events():
     try:
         fullReturnInformation = SystemRequestHandler.get_all_return_information(
             request.get_json())
-    except ApiCallNonResposive:
-        fullReturnInformation["error"] = "API did not call"
+    except ApiCallNonResposive as error:
```

Comment:
We can put all these exceptions into a tuple (maybe even in the exceptions.py file. and write except allCustomExceptions (blah blah blah). This will allow us to not duplicate code like this because regardless of the try outcome, the except is the same.